apps/src/blockly/addons/contextMenu.js

```diff
@@ -173,11 +173,12 @@ const registerDarkMode = function () {
       return MenuOptionStates.ENABLED;
     },
     callback: function (scope) {
+      const currentTheme = scope.workspace?.getTheme();
```

Comment:
Can `scope.workspace` be null? (asking because of the `?`) If it can, we will hit a null reference in `currentTheme.name`, and if `name` could be null we will hit one with `baseName()`, which calls `themeName.replace`

Reply:
Good question. I actually don't think it's possible for `scope.workspace` to be null because this menu option's scope type is workspaces (we can only use this option on a  workspace, not e.g. a block). I don't think we actually need the `?`. Given this explanation, is your concern resolved?

Reply:
yep!